src/bank02_player_menus.asm

```diff
@@ -5812,14 +5826,13 @@ drawManaOnStatusBar:
 
 drawMoneyOnStatusBar:
```

Comment:
You recommended moving `drawNumberAndSymbolOnStatusBar` to the expansion code bank, but I am thinking `drawMoneyOnStatusBar` should be moved as well, especially because it already has a trampoline.

1. Rename `drawMoneyOnStatusBarTrampoline` to `drawMoneyOnStatusBar_trampoline` (as long as it's being edited).
2. Change the single all of `drawMoneyOnStatusBar` to use `drawMoneyOnStatusBar_trampoline` instead.
3. Change `jp_to_bank 02, drawMoneyOnStatusBar` to `jp_to_bank 11, drawMoneyOnStatusBar`
4. Copy both routines to `expansion_code.asm`
5. Add `call_to_bank_target drawMoneyOnStatusBar` to the jump table at the beginning of `expansion_code.asm`
6. Don't forget to copy `clearStatusBarSection` to `expansion_code.asm` as well (oops)
8. Change the names of the copied functions so they don't conflict. (I named them with _2 but it might be better to name the new ones _moved or something?)
9. Build to test that I'm not missing anything
10. Restore bank 2 so the moved functions match the unmodified original rom.
11. Any cleanup or combining you want to do now that `clearStatusBarSection` and `drawNumberAndSymbolOnStatusBar` only have one caller and no space restrictions?

If you don't want to do this, I certainly can (I've done it once before, so I'm a pro!), but it's pretty easy and would be a good intro to the trampoline system.